osbuild.spec

```diff
@@ -70,6 +72,10 @@ install -p -m 0755 $(find sources -type f) %{buildroot}%{pkgdir}/sources
 mkdir -p %{buildroot}%{pkgdir}/stages/osbuild
 mkdir -p %{buildroot}%{pkgdir}/assemblers/osbuild
 
+# documentation
+install -d %{buildroot}%{_mandir}/man1
```

Comment:
The stuff above uses `mkdir`, now you use `install -d`. I am unsure what the Fedora policy is, but maybe keep consistency? I would actually prefer a full-fledged `install -d -m 755`, but maybe it is unrelated to this PR.

Reply:
Good point. Had a very quick look and could not find anything in the guidelines itself. The [example](https://docs.fedoraproject.org/en-US/quick-docs/creating-rpm-packages/index.html) uses `mkdir -p` at one point (step 7):
```
mkdir -p %{buildroot}/%{_bindir}
install -p -m 755 %{SOURCE0} %{buildroot}/%{_bindir}
```

Reply:
Changed to `mkdir -p`